sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -77,7 +77,12 @@
     </template>
     <notice-of-assessment-form-view
       :assessmentId="assessmentId"
+      :canAcceptAssessment="canAcceptAssessment"
+      :currentAssessmentId="currentAssessmentId"
+      :viewOnly="viewOnly"
       @assessmentDataLoaded="assessmentDataLoaded"
+      @confirmCancelApplication="confirmCancelApplication"
+      @confirmAssessment="confirmAssessment"
```

Comment:
The buttons do not need to be added to the shared component since it is a specific student feature.
Adding it here would also prevent the need for extra properties.

Reply:
The properties no longer should be provided for `notice-of-assessment-form-view`.